azext_iot/_params.py

```diff
@@ -343,9 +343,8 @@ def load_arguments(self, _):
                          help='Maximum number of configurations to return.')
 
     with self.argument_context('iot edge') as context:
-        context.argument('config_id', options_list=['--deployment-id', '-d', '--config-id', '-c'],
-                         help="Target deployment name. Option '--config-id' has been deprecated and will be "
-                         "removed in a future release. Use '--deployment-id' instead.")
+        context.argument('config_id', options_list=['--deployment-id', '-d'],
```

Comment:
since --config-id is the one being removed, should we change context.argument('config_id', options_list=['--deployment-id', '-d'] to context.argument('deployment_id', options_list=['--deployment-id', '-d']?

Reply:
Both edge deployments and ADM ultimately share the same API's, so API wise its a "config Id" ...but no issues wanting to update to the identifier here. We wanted to separate --deployment-id/--config-id for the different edge deployment/hub configuration command spaces (from the perspective of the user).